.travis.yml

```diff
@@ -2,8 +2,14 @@ language: python
 python:
   - "2.7"
   - "3.4"
+env:
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+  - DJANGO_VERSION=1.9b1
 install:
   - pip install -r dev-requirements.txt
+  - pip uninstall django --yes
```

Comment:
we could do that but then we've to update documentation. i thought it's much easier. at least for now, until we introduce more other similar cases.